packages/exp/src/commands/generate-unimodule.js

```diff
@@ -0,0 +1,111 @@
+/**
+ * @flow
+ */
+
+export default (program: any) => {
+  program
+    .command('generate-unimodule')
+    .alias('gum')
+    .description('Generate new unimodule.')
+    .asyncAction(async () => {
+      const readline = require('readline');
+      const proc = require('child_process');
+      const path = require('path');
+      const npmVersionOfTemplate = '1.0.1';
+
+      const rl = readline.createInterface({
+        input: process.stdin,
+        output: process.stdout,
+      });
+
+      const askQuestionAsync = (question, resultMerger) =>
+        new Promise((resolve, reject) => {
+          rl.question(`${question} `, answer => {
```

Comment:
Please use the function in [`prompt.js`](https://github.com/expo/expo-cli/blob/master/packages/exp/src/prompt.js) that returns a promise and also handles non-interactive mode.